.github/workflows/master.yml

```diff
@@ -10,6 +10,10 @@ on:
       - master
       - branch-[0-9]+.[0-9]+
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.pull_request == 'pull_request' }}
```

Comment:
Canceling runs on the master branch makes it difficult to detect issues. This line prevents that.

Reply:
We should NOT cancel runs on the master branch to make it easier to identify a PR that breaks our CI checks.